src/index.html

```diff
@@ -8,6 +8,97 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="star starfirst">
```

Comment:
```suggestion
      <div class="star">
```
These star elements should have only one class